src/main/java/org/fundacionjala/coding/oscar/Kata6.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.coding.oscar;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class Kata6 {
+    private static final int MULT = 3;
+
+    /**
+     * This method validate the number.
+     *
+     * @param stringNumber is the number in a string.
+     * @return true if the checksum is equal at the last digit otherwise return false.
+     */
+    public boolean validate(String stringNumber) {
+        int sum = 0;
+        for (int index = 0; index < stringNumber.length() - 1; index++) {
+            if (index % 2 == 0) {
+                sum += Integer.parseInt(stringNumber.substring(index, index + 1));
+            } else {
+                sum += MULT * Integer.parseInt(stringNumber.substring(index, index + 1));
+            }
```

Comment:
this code can be refactor in two lines of code using a ternary operator.
something like this.

```java
int number = Integer.parseInt(stringNumber.substring(index, index + 1));
sum += index % 2 ? number : number * MULT;
```